src/public/modules/forms/admin/directives/settings-form.client.directive.js

```diff
@@ -143,9 +144,11 @@ function settingsFormDirective(
           )
         }
 
-        $scope.isDisableAuthType = () => {
+        $scope.isDisableAuthType = (authType) => {
           return (
             $scope.isFormPublic() ||
+            (authType.val === 'SGID' &&
+              !BetaService.userHasAccessToFieldType($scope.user, 'sgid')) ||
```

Comment:
this implementation looks like if the beta flag is not enabled, the user can see the SGID option but it is disabled for them right? might this be confusing for users who see a new option but don't know why it's disabled for them?

Reply:
It's possible, but this may be deliberate; if they send in a support query about it, FormSG can then direct that query to sgID.